sklearn/preprocessing/data.py

```diff
@@ -111,10 +120,14 @@ def scale(X, axis=0, with_mean=True, with_std=True, copy=True):
 
     To avoid memory copy the caller should pass a CSC matrix.
 
+    See examples/preprocessing/plot_all_scaling.py for a comparison of the
```

Comment:
Is it better to make this a sphinx reference? Does it make sense to place it under Examples rather than Notes?